Tests/StreamChatTests/Repositories/ConnectionRepository_Tests.swift

```diff
@@ -470,7 +470,7 @@ final class ConnectionRepository_Tests: XCTestCase {
         XCTAssertEqual(result?.value, connectionId)
     }
 
-    func test_connectionId_doesNotDeadlock() {
+    func test_connectionId_triggersCompletions_whenConcurrentlyCalled() {
```

Comment:
Just renamed to match with naming to a similar test in [PR 2986](https://github.com/GetStream/stream-chat-swift/pull/2986)